pydrive2/auth.py

```diff
@@ -347,6 +348,11 @@ def _InitializeStoragesFromSettings(self):
                     "Please specify credentials file to read"
                 )
             result[backend] = Storage(credentials_file)
+        elif backend == "dictionary":
+            result[backend] = DictionaryStorage(
```

Comment:
`result = {"file": None}` - check this line above, it might be needed by some logic to have that default dict, might need to add `dictionary` there